core/src/main/python/wlsdeploy/util/cla_helper.py

```diff
@@ -422,6 +426,31 @@ def persist_model(model_context, model_dictionary):
         model_translator.PythonToFile(model_dictionary).write_to_file(model_file.getAbsolutePath())
 
 
+def load_properties_file(current_arg_dict, exception_type=None):
+    wls_deploy_path = os.path.join(os.environ.get(path_utils.WLSDEPLOY_HOME_VARIABLE, ''), 'lib', TOOL_PROPERTIES_FILE_NAME)
```

Comment:
This should call path_utils.find_config_path(TOOL_PROPERTIES_FILE_NAME), so that the WDT_CUSTOM_CONFIG variable can be used to put the file outside the WLSDEPLOY install.
(it prepends WLSDDEPLOY/lib if the variable is not set)

Reply:
changed

Reply:
Done